### ISSUE TITLE
Improve image filename enumeration

### ISSUE THREAD

**Issue by sassanostvar** [napari/napari-animation#139]
Hi All, 

This is hardly critical, but it would be helpful to add more digits/leading zeros to enumerated output filenames. This would make it easier to preserve the right sequential order in downstream command-line applications like ImageMagick's convert. 

One solution is to change line 218 in [https://github.com/napari/napari-animation/blob/main/napari_animation/animation.py]: 

from 

```
fname = folder_path / (
                    path_obj.stem + "_" + str(ind) + ".png"
)
```

to 

```
fname = folder_path / (
                    path_obj.stem + "_" + str(ind).zfill(5) + ".png"
)
```

Thanks!

**Comment by alisterburt** [comment]
Hi there!

Really great suggestion, thanks for taking the time to suggest the improvement 🙂

Would you like to submit a pull request with the changes you proposed? I can help you through the process if it's new to you - this way you would become part of the contributors list 

Also fine if you would rather I integrate the changes!

**Comment by sassanostvar** [comment]
Great, thank you! I just tried to submit a draft PR. I hope it worked?
Really appreciate your time and all the great work!

**Comment by alisterburt** [comment]
Great work @sassanostvar ! I just merged #140 and pushed a new tag so the changes should be available in v0.0.4 on PyPI soon

https://github.com/napari/napari-animation/actions/runs/3451585929
